src/main/java/use_cases/PostManager.java

```diff
@@ -0,0 +1,33 @@
+package use_cases;
+
+import java.util.ArrayList;
+
+/*
+store all the posts existing in the app,
+add a new post, comment/like a Post
+ */
+public class PostManager {
+    // TODO: Change String to Post when the entity is created
+    private ArrayList<String> posts;
+
+    public PostManager(){
+        this.posts = new ArrayList<String>();
```

Comment:
What are these Strings? How would they be used

Reply:
It is just for temporal type, my plan is changing it into Post so that we we could really store the posts we want